src/main/java/com/zufar/onlinestore/payment/converter/StripePaymentMethodConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.payment.converter;
+
+import com.stripe.model.Token;
+import com.stripe.param.PaymentMethodCreateParams;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+import org.mapstruct.Named;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
```

Comment:
maybe better use "spring" in the componentModel instead of MappingConstants.ComponentModel.SPRING ?